server/data/data-library/src/main/java/org/apache/james/rrt/lib/MappingImpl.java

```diff
@@ -33,6 +33,7 @@
     private static final long serialVersionUID = 1L;
 
     private static final String ADDRESS_PREFIX = "";
+    private boolean address;
```

Comment:
all the changes of this file are unrelated too

Reply:
fixed